enhancements/mcd-reboot-dictionary.md

```diff
@@ -0,0 +1,275 @@
+of unnecessary periods of degraded system and application availability.
+
+#### Story 4
+
+As an admin, I want to continue to force reboots for network configuration 
+updates and changes to kernel flags, so that they are applied in a timely 
+manner.
+
+### Implementation Details/Notes/Constraints [optional]
+
+This is an interim solution designed to quickly address a market need and
+gather data for a long term approach.
+
+### Risks and Mitigations
+
+The risks are of false positives (an update that requires a reboot does not
+trigger one), and false negatives (an update that does not require a reboot
+triggers one anyway).  While the latter may be unexpected by an admin that
+knows of this feature, at worst it is a performance issue and cannot be
+considered a regression over the current behaviour.
+
+False negatives can be mitigated by defaulting to the current behaviour
+(reboot), preventing the admin from tampering with the whitelist, and only
+adding entries to the whitelist after Engineering and QE validate that no
+possible contents of the file could require a reboot.
+
+## Design Details
+
+### Test Plan
+
+- Missing and empty whitelist
+- Changes to files not on the whitelist
+- Changes to files on the whitelist requiring no action
+- Changes to files on the whitelist requiring a service restart
+- Changes to files on the whitelist with invalid actions/data
+- Changes to files on the whitelist with valid actions/data but the action fails or takes too long
+- Large whitelist performance
+
+### Graduation Criteria
+
+##### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- Whitelist format stability
+- Sufficient test coverage
+- Provisional whitelist contents
+
+##### Tech Preview -> GA 
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- A supportable whitelist
+
+##### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+As per the current MCD strategy.
+
+### Version Skew Strategy
+
+If the MCD does not contain this feature, or only knows about the old
+whitelist, then the first change that affects a new entry will trigger a reboot
+(which can not be a regression) and subsequent changes will be acted on with
+the new list.  Any changes affecting entries that have been removed will have
+the old behaviour (also not a regression).
+
+If the MCD knows about a newer whitelist than the rest of the system, then
+there is a risk that a reboot will not occur for a version of the component
+that still requires it.  This could be mitigated by ensuring the adding a file
+to the whitelist always happens N versions after any engineering work that was
+required to make it possible.
+
+## Implementation History
+
+- Initial version: 2019-12-15
+
+## Drawbacks 
+
+The idea is to find the best form of an argument why this enhancement should _not_ be implemented.
```

Comment:
Since this enhancement is advocating for a very specific approach, I think the drawbacks to such an approach should be articulated